### PR TITLE
Add libraries

### PR DESCRIPTION
**Pull request by planetis-m**

### CODE REVIEW

**Review by ringabout — COMMENTED** [review]
Awesome! The newly created heading should be added to [the index](https://github.com/ringabout/awesome-nim#-contents-)

**Comment by planetis-m** [comment]
@ringabout done!

File: README.md [hunk 1]
```diff
@@ -335,10 +337,17 @@ Nim provides unique features for seamless and transparent interoperability with
 - [protobuf-nim](https://github.com/PMunch/protobuf-nim) - Protobuf implementation in pure Nim that leverages the power of the macro system to not depend on any external tools.
 - [flatty](https://github.com/treeform/flatty) - Tools and serializer for plain flat binary files.
 - [nesm](https://xomachine.gitlab.io/NESM/) - NESM is a tool that generates serialization and deserialization code for a given object.
+- [bingo](https://github.com/planetis-m/bingo) - Binary serialization framework for Nim
```
Comment (ringabout) [inline README.md:340]:
```suggestion
- [bingo](https://github.com/planetis-m/bingo) - Binary serialization framework for Nim.
```

File: README.md [hunk 2]
```diff
@@ -335,10 +337,17 @@ Nim provides unique features for seamless and transparent interoperability with
...
 
 
 ## Text
 
+### String Types
+
+- [ssostrings](https://github.com/planetis-m/ssostrings) - Small String Optimized (SSO) string implementation 
+- [cowstrings](https://github.com/planetis-m/cowstrings) - Copy-On-Write string implementation according to nim-lang/RFCs#221
```
Comment (ringabout) [inline README.md:348]:
```suggestion
- [ssostrings](https://github.com/planetis-m/ssostrings) - Small String Optimized (SSO) string implementation.
- [cowstrings](https://github.com/planetis-m/cowstrings) - Copy-On-Write string implementation according to nim-lang/RFCs#221.
```

File: README.md [hunk 3]
```diff
@@ -705,6 +715,12 @@ Nim provides unique features for seamless and transparent interoperability with
 - [unittest2](https://github.com/status-im/nim-unittest2) - Fork of the "unittest" Nim module focusing on parallel test execution, test-level scoping and strict exception handling.
 
 
+### Fuzzing
+
+- [drchaos](https://github.com/status-im/nim-drchaos) - A powerful and easy-to-use fuzzing framework in Nim for C/C++/Obj-C targets 
+- [libfuzzer](https://github.com/planetis-m/libfuzzer) - LibFuzzer's interface bindings
```
Comment (ringabout) [inline README.md:721]:
```suggestion
- [drchaos](https://github.com/status-im/nim-drchaos) - A powerful and easy-to-use fuzzing framework in Nim for C/C++/Obj-C targets.
- [libfuzzer](https://github.com/planetis-m/libfuzzer) - LibFuzzer's interface bindings.
```

**Comment by ringabout** [comment]
LGTM, thanks!

**Pull request merged**